packages/cypress/package.json

```diff
@@ -54,6 +54,6 @@
     "ws": "^8.14.2"
   },
   "peerDependencies": {
-    "cypress": ">=5.3.0"
+    "cypress": "^13"
```

Comment:
Cypress 13 was released almost one year ago (08/29/2023). We don't have any tests for this whole integration, trying to test compatibility manually with an extended range of versions is a nightmare. I don't feel we can quite guarantee compatibility right now with other versions, it's impractical.

If requested by clients we can always loosen up this range after manually confirming that a particular combination works. We'll release a new major of this package soon so I think we should grab the opportunity and bump this here.

Note that supporting more right now would require some extra work here, I changed this because I ran into typing problems (that likely were related to breaking changes in Cypress).

Reply:
I support this.